src/instruments/src/SD/Pages/Crz/Crz.tsx

```diff
@@ -0,0 +1,301 @@
+import ReactDOM from 'react-dom';
+import React, { useEffect, useState } from 'react';
+import { getRenderTarget, setIsEcamPage } from '../../../Common/defaults';
+import { SimVarProvider, useSimVar } from '../../../Common/simVars';
+import { NXDataStore } from '../../../Common/persistence';
+import { splitDecimals, valueRadianAngleConverter, polarToCartesian } from './common';
+
+import './Crz.scss';
+
+setIsEcamPage('crz_page');
+
+export const CrzPage = () => (
+    <>
+        <svg id="crz-page" viewBox="0 0 600 600" style={{ marginTop: '-60px' }} xmlns="http://www.w3.org/2000/svg">
+            <text className="Title" x="300" y="20">CRUISE</text>
+
+            <text className="SubTitle" x="50" y="60">ENG</text>
+            <FuelComponent />
+            <OilComponent />
+
+            <text className="SubTitle" x="50" y="330">AIR</text>
+            <PressureComponent />
+
+            <CondComponent />
+
+        </svg>
+    </>
+);
+
+export const FuelComponent = () => {
+    const unit = parseFloat(NXDataStore.get('CONFIG_USING_METRIC_UNIT', '1'));
```

Comment:
You can use the `usePersistentProperty` hook here:

https://github.com/flybywiresim/a32nx/blob/bbb804a8bd43180830fd97b3996ef1df48785fe8/src/instruments/src/Common/persistence.tsx#L41

Reply:
Done